distribution/org.wso2.developerstudio.eclipse.distribution.project/src/org/wso2/developerstudio/eclipse/distribution/project/ui/wizard/ProjectImportWizard.java

```diff
@@ -70,8 +75,10 @@ public boolean performCancel() {
 	 */
 	private void searchAndRemoveGraphicalSynapseCongif(IProject project) {
 		try {
-			if (project.hasNature("org.wso2.developerstudio.eclipse.esb.project.nature")) {
+			if (project.hasNature(Constants.ESB_PROJECT_NATURE)) {
 				deleteGraphicalSynapseConfigDir(project);
+			} else if (project.hasNature(Constants.GENERAL_PROJECT_NATURE)) {
+				deleteGraphicalSynapseConfigFiles(project);
 			}
 		} catch (CoreException e) {
 			log.error("Error while deleting graphical-synapse-config directory in " + project);
```

Comment:
Would it be better to include the error stack trace in the logs here ? (also i agree this is not a part of the changes you've made)